docs/sophia_stdlib.md

```diff
@@ -2264,3 +2265,125 @@ accept arbitrary `frac`s from the surface you should report it as a
 
 If you expect getting calls with malformed `frac`s in your contract, you should use
 this function to verify the input.
+
+## Set
+
+### Types
+
+```
+record set('a) = { to_map : map('a, unit) }
+```
+
+### Functions
+
+#### new
+
+```
+Set.new() : set('a)
+```
+
+Returns an empty set
+
+#### member
+
+```
+member(e : 'a, s : set('a)) : bool
+```
+
+Checks if the element is present in the Set
```

Comment:
```suggestion
Checks if an element is present in the set
```
I think it works like that. Tbh, not sure – articles aren't part of my mother-tongue (sadly)

Reply:
Made some changes here 588684ba16cb8e6bc33b30f7fcd54f184ff02285